src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -176,3 +176,11 @@ ascanalpha.traceaxdscanner.otherinfo = Based on response status code Trace Viewe
 ascanalpha.traceaxdscanner.refs = https://msdn.microsoft.com/en-us/library/bb386420.aspx\nhttps://msdn.microsoft.com/en-us/library/wwh16c6c.aspx\nhttps://www.dotnetperls.com/trace
 ascanalpha.traceaxdscanner.soln = Consider whether or not Trace Viewer is actually required in production, if it isn't then disable it. If it is then ensure access to it requires authentication and authorization.
 
+ascanalpha.testnosqlinjection.name=NoSQL Injection
+ascanalpha.testnosqlinjection.desc=NoSQL injection may be possible.
+ascanalpha.testnosqlinjection.soln=Do not trust client side input and escape all data on the server side. \n Avoid to use the input type directly into the mongoDB where and group clauses and upgrade the mongoDB drivers at the last available.
+ascanalpha.testnosqlinjection.refs=https://arxiv.org/pdf/1506.04082.pdf \n https://blog.sqreen.io/mongodb-will-not-prevent-nosql-injections-in-your-node-js-app/?utm_content=buffer25729&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer
```

Comment:
Spaces should be removed.
The tracking parameters on the second URL should be removed.